hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -972,24 +992,26 @@ private static FileStatusAdapter getFileStatusAdapterForVolume(
    * Generate a FileStatusAdapter for a bucket.
    * @param ozoneBucket OzoneBucket object.
    * @param uri Full URI to OFS root.
+   * @param owner Owner of the parent volume of the bucket.
+   * @param group Group of the parent volume of the bucket.
    * @return FileStatusAdapter for a bucket.
    */
   private static FileStatusAdapter getFileStatusAdapterForBucket(
-      OzoneBucket ozoneBucket, URI uri, String username) {
+      OzoneBucket ozoneBucket, URI uri, String owner, String group) {
     String pathStr = uri.toString() +
         OZONE_URI_DELIMITER + ozoneBucket.getVolumeName() +
         OZONE_URI_DELIMITER + ozoneBucket.getName();
     if (LOG.isDebugEnabled()) {
-      LOG.debug("getFileStatusAdapterForBucket: ozoneBucket={}, pathStr={}, "
-              + "username={}", ozoneBucket.getVolumeName() + OZONE_URI_DELIMITER
-              + ozoneBucket.getName(), pathStr, username);
+      LOG.debug("getFileStatusAdapterForBucket: ozoneBucket={}, pathStr={}",
+          ozoneBucket.getVolumeName() + OZONE_URI_DELIMITER +
+              ozoneBucket.getName(), pathStr);
     }
     Path path = new Path(pathStr);
     return new FileStatusAdapter(0L, path, true, (short)0, 0L,
         ozoneBucket.getCreationTime().getEpochSecond() * 1000, 0L,
-        FsPermission.getDirDefault().toShort(),  // TODO: derive from ACLs later
-        // TODO: revisit owner and group
-        username, username, path, new BlockLocation[0]);
+        FsPermission.getDirDefault().toShort(),
+        // TODO: maybe derive owner and group from ACLs later
```

Comment:
NIT: The comment is not accurate. The ACLs will not be able to provider owner and group information. We might add owner information as bucket metadata in future.

Reply:
Thanks! Removed the comment.